ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,156 @@
-- {"app":"core","message":"Login failed: 'admin' (Remote IP: '127.0.0.1', X-Forwarded-For: '')","level":2,"time":"2015-06-09T08:16:29+00:00","@source":"NextCloud"}
-- {"reqId":"5576a04643d8e","app":"core","message":"Login failed: 'admin' (Remote IP: '127.0.0.1', X-Forwarded-For: '')","level":2,"time":"2015-06-09T08:13:58+00:00","method":"POST","url":"\/NextCloud\/index.php","@source":"NextCloud"}
-- {"app":"core","message":"Login failed: user 'admin' , wrong password, IP:127.0.0.1","level":2,"time":"2015-06-09T08:10:29+00:00","@source":"NextCloud"}
-- {"reqId":"55769fcacd1e0","app":"core","message":"Login failed: user 'admin' , wrong password, IP:127.0.0.1","level":2,"time":"2015-06-09T08:11:54+00:00","method":"POST","url":"\/NextCloud\/index.php","@source":"NextCloud"}
-- {"reqId":"4ETnKW0UyDBNmL4z\/umV","remoteAddr":"127.0.0.1","app":"PHP","message":"Redis::connect(): connect() failed: No such file or directory at \/var\/www\/NextCloud\/lib\/private\/RedisFactory.php#60","level":3,"time":"2017-08-21T16:00:34+02:00","method":"PROPFIND","url":"\/remote.php\/dav\/addressbooks\/users\/admin\/example\/","user":"admin","@source":"NextCloud"}
-- {"reqId":"4j2DKpvOh0OezXVwfuLO","remoteAddr":"127.0.0.1","app":"PHP","message":"fopen(\/var\/www\/NextCloud\/data\/user 1\/thumbnails\/1234\/32-32.png): failed to open stream: No such file or directory at \/var\/www\/NextCloud\/lib\/private\/Files\/Storage\/Local.php#278","level":3,"time":"2017-07-15T23:59:20+02:00","method":"GET","url":"\/core\/preview.png?file=%2Fexample.txt&c=123&x=32&y=32&forceIcon=0","user":"user 1","@source":"NextCloud"}
-- Examples syslog:
-- Sep  1 20:16:09 foo NextCloud[15463]: {core} Login failed: 'test' (Remote IP: '127.0.0.1')
-- Sep  1 22:16:33 foo NextCloud[15467]: {core-preview} Passed filename is not valid, might be malicious (file:"test";ip:"127.0.0.1","@source":"NextCloud")
--->
-
 <group name="json,nextcloud,">
 
   <rule id="88200" level="0">
     <decoded_as>json</decoded_as>
     <field name="@source">NextCloud</field>
-    <description>NextCloud messages grouped.</description>
     <options>no_full_log</options>
+    <description>NextCloud messages grouped.</description>
   </rule>
 
   <rule id="88201" level="0">
     <decoded_as>nextcloud</decoded_as>
-    <description>NextCloud messages grouped.</description>
     <options>no_full_log</options>
+    <description>NextCloud messages grouped.</description>
   </rule>
 
   <rule id="88202" level="1">
     <if_sid>88200,88201</if_sid>
     <match>Login attempt: </match>
-    <description>NextCloud authentication attempt.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
     <options>no_full_log</options>
+    <description>NextCloud authentication attempt.</description>
+    <group>gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
Why are we dropping pci_dss_10.2.5 (Use and modification of identification and authentication mechanisms)? I would say it applies here.

Reply:
Agree, reincluding 10.2.5

Reply:
Done